doc/nrf/protocols/matter/end_product/certification.rst

```diff
@@ -330,3 +332,26 @@ This variant is meant for Matter components that use Matter software that was al
    :alt: Matter certification by similarity overview
 
    Matter certification by similarity overview
+
+.. _ug_matter_device_certification_matter_samples:
+
+Certification status of Matter samples
+**************************************
+
+The |NCS| includes several :ref:`matter_samples` that are example implementations of Matter devices.
+The samples are maintained to fulfill Matter certification requirements as much as possible, however they did not get official Matter compliance certificates.
+You can use them as a reference for creating an own application, however it does not guarantee that the application will pass the Matter certification.
+
+The Matter certification program currently does not provide any form of platform certification, so the product maker is solely reponsible for ensuring that the application used fulfills all Matter certification requirements.
+Before you start applying for the Matter certification, you should make sure that the application configuration generated using :file:`.zap` file is compatible with the PICS generated for the certification purpose.
+You can find the information about the specification and how to select PICS for you product on the `CSA Matter Resource Kit`_ page, in the `Specification` section.
+Especially, make sure that:
```

Comment:
I will even inform to check data model for each cluster, to verify 'global attributes' so:
Feature Map
Cluster Revision
Attribute List
Accepted Command List
Generated Command List
Event List (this is currently not supported even for future Matter 1.2)

Reply:
Added 3rd bullet point including all the list attributes.